### PR TITLE
Partly fixes #300: Docs: Addressed issues in chapters 1 to 4

### LINKED ISSUE #300: Final review of docs and help texts for fixes and editorial changes
Done:
* #351 - chapters 1 to 4
* #356 - chapter 5 class commands; some terms in appendix
* #361 - appendix (some terms)
* #359 - chapter 5 instance & qualifier commands
* #362 - chapter 5 server commands
* #364 - chapter 5 connection commands, help, repl
* chapter 7 (mock support) - nothing to do
* chapter 8 (development) - nothing to do
* #376 - chapter 9 (appendix)
* #377 - help texts of all commands

### PR DESCRIPTION
**Pull request by andy-maier**
Covers chapters 1 to 4.
Ready for review and merge.

### CODE REVIEW

File: docs/pywbemcligeneraloptions.rst [hunk 1]
```diff
@@ -580,49 +612,49 @@ PYWBEMCLI_MOCK_SERVER           ``--mock-server``
 PYWBEMCLI_LOG                   ``--log``
 ==============================  ============================
 
-If any of these environment variables are set, the corresponding general
-options on the command line are not required and the value of the environment
-variable is used. If both the environment variable and the command line option
-are included the command line option overrides the environment variable with no
+If these environment variables are set, the corresponding general options
+default to the value of the environment variables.
+
```
Comment (KSchopmeyer) [inline docs/pywbemcligeneraloptions.rst]: We have several one sentence paragraphs here.  Lets group the first 3 sentences into a single paragraph.

File: docs/pywbemcligeneraloptions.rst [hunk 2]
```diff
@@ -68,59 +74,48 @@ interactive mode and as a table:
...
-   server.
+   The mock WBEM server is part of pywbemcli and allows testing or
+   demonstrating pywbemcli without having access to a real WBEM server.
+   For details, see :ref:`Mock WBEM server support`.
 
-3. Define connections for either a WBEM server or mock WBEM server with
-   pywbemcli and add the data to a pywbemcli :term:`connections file`.
-   pywbemcli can access the server data by simply defining the name on with the
-   pywbem call.
+3. Refer to a persisted connection definition for either a WBEM server or
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: See below.

File: docs/pywbemcligeneraloptions.rst [hunk 3]
```diff
@@ -68,59 +74,48 @@ interactive mode and as a table:
...
+   mock WBEM server by using the :ref:`--name general option` that
+   specifies the name of the connection definition.
+
+   Persisted connection definitions are stored in a :term:`connections file`
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: I tried but it makes the first paragraph a more complex sentence. I prefer making the statements in two simpler sentences.

File: docs/pywbemcligeneraloptions.rst [hunk 4]
```diff
@@ -271,59 +279,61 @@ connections.
...
+WBEM server.
+
+If this option is not specified, the default namespace will be ``root/cimv2``.
+
+The default namespace will be used if the ``--namespace``/``-n`` command
+option is not used on a command.
 
-This is the namespace used on all server operation requests unless a specific
-namespace is defined by:
+Note that the ``class find`` command searches in all namespaces, if its
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: DONE

File: docs/pywbemcligeneraloptions.rst [hunk 5]
```diff
@@ -380,12 +392,15 @@ predefined timeout (normally 30 seconds) if this option is not defined.
 --verify general option
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-If the ``--verify\--no-verify`` general option is set ``--no-verify``, the client does not
-verify any certificates received from the WBEM server. If --verify is defined, certificates
-returned from the server are verified. Any certificate returned
-by the WBEM server is accepted. The default is to verify certificates returned.
+The pair of ``--verify`` and ``--no-verify`` general options are a boolean
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: We say "the --xxx general option" all over the place, so talking about them as options (in plural) is consistent with that. We do not talk about option names.

Where it becomes strange is when I tried to describe them as two options that are actually one option. I changed it to "The pair of ... general options control whether or not ...".
DONE?

File: docs/pywbemcligeneraloptions.rst [hunk 6]
```diff
@@ -423,68 +441,75 @@ key file. The client private key should then be part of the file defined by
...
 
 --timestats general option
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This boolean general options enables the gathering and display of time
+The ``--timestats`` general option is a boolean option that enables the
+gathering and display of time
 statistics on the interactions with the WBEM server.  If enabled, the
-timestatistics are output after each command is executed including the
+time statistics are output after each command is executed including the
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: OK

File: docs/pywbemcligeneraloptions.rst [hunk 7]
```diff
@@ -649,22 +681,27 @@ the use of these operations through two general options:
...
+  from experience is a logical choice.
+
+The default alternative ``either`` is probably the most logical setting for
+``--use-pull``, unless you are specifically testing the use of pull
+operations.
 
-There are limitations with using the the ``either`` choice as follows"
+However, there are some limitations with using the ``either`` choice as follows:
 
 * The original operations did not support the filtering of responses with a
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: Not sure how you want this to be changed?

File: docs/pywbemcligeneraloptions.rst [hunk 8]
```diff
@@ -978,72 +1012,74 @@ standard Python logging facility.
...
-For example the following log string logs the pywbem API calls and responses
-summary information to a file and the HTTP requests and responses to stderr:
+For example the following log configuration string logs the pywbem API calls
+and writes summary information to a log file and the HTTP requests and
+responses to stderr:
 
 .. code-block:: text
 
-      $ pywbemcli --log api=file:summary,http=stderr
+    $ pywbemcli --log api=file:summary,http=stderr
```
[the remark this answers is not in the document]
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: DONE

File: docs/pywbemcligeneraloptions.rst (hunk 1 above)
Comment (andy-maier) [reply docs/pywbemcligeneraloptions.rst]: DONE

**Pull request merged**